docs/resource-policy/policy/topology-aware.md

```diff
@@ -517,17 +517,17 @@ metadata:
       type: prefix
       paths:
         - /
-    # Allow hints from /sys/devices/pci*/*d7:00.0/*:d8:00.1 for container ctr1
+    # Allow hints from /sys/devices/pci*/*d7:00.0/*:d8:00.1 for container ctr1.
     allow.topologyhints.resource-policy.nri.io/container.ctr1: |+
       type: glob
       paths:
         - /sys/devices/pci*/*d7:00.0/*:d8:00.*
-    # Allow hints from a special path (which usually should resolve to a block
+    # Allow hints for a local NVME block devices.
```

Comment:
s/for a local/for local

Reply:
Argh... no change is too small for me to typo it. It must be a unique talent.